features/shared/continuing_after_success.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Show clear error if trying to continue after executing a successful command
+
+  As a developer accidentally trying to continue a command after it completed successfully
+  I should see a clear error that the command cannot be continued
```

Comment:
I should see a friendly and descriptive message that the command can not be continued